### PR TITLE
feat : added basic Routes to the pages 

### PR DESCRIPTION
**Pull request by Tanjum786**
added routes to the 

- Home page 
- product page
- wishlist 
- cart 
- Authentication

### CODE REVIEW

File: src/components/Navbar-comp/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <Link to="/">
+          <h1 className="e_website_name">E-Galaxy-store</h1>
+          <h5 className="ui_tagline">Find it, love it, buy it.</h5>
+        </Link>
+      </div>
+      <div className="search">
+        <button>
+          <i className="fa fa-search"></i>
+        </button>
+        <input type="text" placeholder=" Search Your Desier" name="search" />
```
Comment (srejitk) [inline src/components/Navbar-comp/Navbar.jsx]: I think you meant Desire here? Correct the typo.

File: src/components/Navbar-comp/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <Link
+              to="/product"
+              className="product_btn padding_small font_small"
+            >
+              Products
+            </Link>
+          </li>
+          <li>
+            <Link to={linkto} className="login_btn">
+              {auth}
```
Comment (srejitk) [inline src/components/Navbar-comp/Navbar.jsx]: I see paths in other links, please update it with the path if you missed it

File: src/pages/Authentication/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                <span>Forgot password ?</span>
+                </Link>
+              </div>
+              <button type="submit" class="login_btn btn_style">
+                Login
+              </button>
+              <p class="new_account_link font_small">
+                Not a user yet ?
+                <Link to="/signup" class="alert_primary">
+                  Creat account
```
Comment (srejitk) [inline src/pages/Authentication/Login.jsx:54]: Typo here, please correct it.

**Review by srejitk — APPROVED** [review]
Suggested a few minor changes. 
Try to mention which files need to be checked from next review.

File: src/pages/Cart/Cart.jsx [hunk 4]
```diff
@@ -30,6 +22,14 @@ export const Cart = () => {
           TotalAmt={"₹250"}
           cartitem={"1"}
         />
+        <Cardcart
+          cardtitle={"SOM2SOM Women Solid Straight Kurta"}
+          cartimg={image7}
+          cartprice={"Rs.250"}
+          cutprice={"Rs.500"}
+          discount={"50%"}
+          Quantity={"Quantity :"}
```
Comment (rdshinde) [inline src/pages/Cart/Cart.jsx:31]: Just to make it easy.
If you could use the single object value or data and then pass destructured values in it then you would not need to pass these many props. One single object can do that job.

**Review by rdshinde — APPROVED** [review]
Looks good. 
Just a small suggestion, try to keep your PR as specific as possible. Too much refactoring in PR of features may decrease the review quality because it gets difficult to focus on feature file for reviewers.

File: src/pages/Cart/Cart.jsx (hunk 4 above)
Comment (Tanjum786) [reply src/pages/Cart/Cart.jsx:31]: yeah i will change this

**Pull request merged**